test/functional/feature_rbf.py

```diff
@@ -702,17 +702,16 @@ def test_replacement_relay_fee(self):
         assert_raises_rpc_error(-26, "insufficient fee", self.nodes[0].sendrawtransaction, tx.serialize().hex())
 
     def test_fullrbf(self):
-        txid = self.wallet.send_self_transfer(from_node=self.nodes[0])['txid']
-        self.generate(self.nodes[0], 1)
-        confirmed_utxo = self.wallet.get_utxo(txid=txid)
 
+        confirmed_utxo = self.make_utxo(self.nodes[0], int(2 * COIN))
         self.restart_node(0, extra_args=["-mempoolfullrbf=1"])
+        assert self.nodes[0].getmempoolinfo()["fullrbf"]
 
         # Create an explicitly opt-out transaction
         optout_tx = self.wallet.send_self_transfer(
             from_node=self.nodes[0],
             utxo_to_spend=confirmed_utxo,
-            sequence=SEQUENCE_FINAL,
+            sequence=BIP125_SEQUENCE_NUMBER + 1,
```

Comment:
Is there an explanation why the arithmetic is easier to understand than a named constant?

Reply:
Not extremely opinionated about this, but I interpret SEQUENCE_FINAL as primarily for disabling locktime, and the fact that it doesn't signal rbf is a side effect. bip125signal + 1 literally means no signal. But fine with whatever is conventionally used / easiest to understand.

Reply:
Ah right. I wonder why `BIP125_SEQUENCE_NUMBER` isn't called `MAX_BIP125_RBF_SEQUENCE`?